hints/darwin.sh

```diff
@@ -357,8 +357,7 @@ EOM
         esac
     fi
 
-    # The OS is buggy with respect to this.
```

Comment:
commit message, perhaps:

Make sure *it* is executed while a mutex locks out other accesses.

Perhaps also "has *a* race" in the subject

Reply:
done